README.md

```diff
@@ -85,79 +85,90 @@ The task folder in this repo holds an existing site, including multiple html fil
     ```bash
     npm run start:watch
     ```
 
     Load up the server and go to localhost:3000 in your browser; expect to see a 404 error. We are now going to serve up the ```home.hbs``` file.
 
-7. Create a ```home.js``` in src/controllers
+8. Create a ```home.js``` in `./src/controllers`
 
-8. In ```home.js``` add the following code:
+9. In ```home.js``` add the following code:
     ```js
     exports.get = (req, res) => {
       res.render('home');
     };
     ```
     Here ```res.render``` renders a view and sends a rendered HTML string to the client.
 
-9. In src/controllers/index.js require in ```home.js```.
+10. In `./src/controllers/index.js` require in ```home.js```.
 
-10. Add the following line after files are required in:
+11. Add the following line after files are required in:
     ``` js
     router.get('/', home.get);
     ```
 
 ### Split up the contents into partials
 We are now going to create partials to simplify our website. Partials allow us to reuse snippets of HTML preventing duplication.
-1. In src/views/partials create files for the partials that you would like and cut the appropriate content into the partial. Consider putting anything into the partials that is repeated across multiple pages and might be used in multiple layouts.
+1. In `./src/views/partials` create files for the partials that you would like and cut the appropriate content into the partial. Consider putting anything into the partials that is repeated across multiple pages and might be used in multiple layouts.
+
+    For example, you might create partial/htmlHead, and insert the html header into it:
+
+    ```hbs
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      <link rel="stylesheet" href="/css/styles.css">
+      <title>Fruit Market</title>
+    </head>
+    ```
 
-    For example, you might create partial/htmlHead, and insert the html header into it. 
-    
     *What else can you see that could be used across multiple layouts?*
 
-3. The next stage is to continue creating the layout in ```main.hbs```. The goal is to have a layout that you can use for all pages on this site except error pages. **Tips**:
+2. The next stage is to continue creating the layout in ```main.hbs```. The goal is to have a layout that you can use for all pages on this site except error pages. **Tips**:
     * Use the [docs](http://handlebarsjs.com/) to find the syntax for inserting partials.
-    * As well as partials you can also place things such as ```<html>```, ```<body>```, ```<main>```, and ```<script>``` tags in here.
     * Make sure you keep the ```{{body}}```.
     * After finishing the main layout, ```home.hbs``` should just include content specific to the page (e.g. the content inside ```<section>``` tags).
```

Comment:
Is this point still valid after the reshuffle?

Reply:
I'll move that point to just after students paste 'home' content into `home.hbs`